Docs: Refer to product docs in whats new for alerting templating feature (#30652)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v7-4.md

### Alert notification query label interpolation

> You can now provide detailed information to alert notification recipients by injecting alert label data as template variables into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields using the `${Label}` syntax. The alert label data is automatically injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated. For more information, refer to the [alert notification docs]({{< relref "../alerting/notifications.md#notification-templating" >}}). 

Reviewer: Please put the link sentence below the image. This will make it consistent with the other entries.

> You can now provide detailed information to alert notification recipients by injecting alert label data as template variables into an alert notification. Labels that exist from the evaluation of the alert query can be used in the alert rule name and in the alert notification message fields using the `${Label}` syntax. The alert label data is automatically injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated. For more information, refer to the [alert notification docs]({{< relref "../alerting/notifications.md#notification-templating" >}}). 

Reviewer 2: ```suggestion
You can provide detailed information to alert notification recipients by injecting alert label data as template variables into an alert notification. You can use labels, which exist from the evaluation of the alert query both within the alert rule name and within the alert notification message fields, by using the `${Label}` syntax. The alert label data is automatically injected into the notification fields when the alert is in the alerting state. When there are multiple unique values for the same label, the values are comma-separated. For more information, refer to the [alert notification docs]({{< relref "../alerting/notifications.md#notification-templating" >}}). 
```

Reviewer 2: These sentences are quite dense, so this suggestion relates to breaking up this density. Please vet for technical accuracy.

Reviewer 3: @osg-grafana, these are good suggestions. However, I would argue to keep "You can now..." in the first sentence since it pertains to changes in 7.4.  Also, I do not think we should begin the second sentence with "You can" again ...

@wbrowne, feel free to accept the changes or make them in a separate PR since this PR is specific to adding the link to the product documentation. If needed, we can scrub the words. Thanks!

Author: Thanks for the feedback 😄  This has already been through multiple docs reviews, so I'm not sure what is 100% necessary to change here in this PR. I'd like to merge this fix and tackle remaining issues in a follow-up if necessary

Reviewer 2: Yes, lean is ideal.

My changes relate to breaking up or handling that 25+ word sentence to align with the style guide:
"* Aim for your sentences to be fewer than 25 words. Instead, use smaller complete phrases or change the format, such as using a list.
* Aim for paragraphs to be three sentences or fewer. Make the text more concise, use more headings, or both."
